ui/app/src/modules/Preview/Tools/AssetsPanel.tsx

```diff
@@ -234,7 +238,7 @@ export default function AssetsPanel() {
       const unmount$ = new Subject();
       fromEvent(elementRef.current, 'dragleave').pipe(
         takeUntil(unmount$)
-      ).subscribe((e: any) => {
+      ).subscribe(() => {
         clearTimeout(timeoutRef.current);
         timeoutRef.current = setTimeout(() => {
           clearTimeout(timeoutRef.current);
```

Comment:
Let's give this a try....
```js
useEffect(() => {
  if (dragInProgress) {
    const dragover$ = fromEvent(elementRef.current, 'dragover').pipe(
      tap((e) => {
        e.preventDefault();
        e.stopPropagation();
      })
    );
    const subscription = fromEvent(elementRef.current, 'dragleave').pipe(
      switchMap(() => interval(100).pipe(takeUntil(dragover$))),
      mapTo(false)
    ).subscribe(setDragInProgress);
    return () => {
      subscription.unsubscribe();
    };
  }
}, [dragInProgress]);
```

Reply:
it is working but has a issue with the dragover